File: 项目/OC项目/百思不得姐(oc)/百思不得姐(oc)/Classes/Me(我)/Controller/WebViewController.m
//
//  WebViewController.m
//  百思不得姐(oc)
//
//  Created by 金亮齐 on 2017/5/16.
//  Copyright © 2017年 醉看红尘这场梦. All rights reserved.
//

#import "WebViewController.h"
#import <WebKit/WebKit.h>
@interface WebViewController ()
@property (weak, nonatomic) IBOutlet UIWebView *webView;
@end

@implementation WebViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    //导航条右边按钮
    self.navigationItem.rightBarButtonItem = [UIBarButtonItem itemWithImage:@"comment_nav_item_share_icon" hightImage:@"comment_nav_item_share_icon_click" target:self action:@selector(share)];
    
    //创建一个WKWebView
    WKWebView *webView = [[WKWebView alloc]initWithFrame:CGRectMake(0, 64, ScreenW, ScreenH)];
    self.webView.scrollView.contentInset = UIEdgeInsetsMake(-64, 0, 0, 0);
    
    [self.webView addSubview:webView];
    
    NSURLRequest *request = [NSURLRequest requestWithURL:_url];
    [webView loadRequest:request];
    // Do any additional setup after loading the view from its nib.
}

- (void)share{
    LogFunc
}

@end

File: 项目/Swift项目/ofo(小黄)/ofo(小黄)/webViewController.swift
//
//  webViewController.swift
//  ofo(小黄)
//
//  Created by 金亮齐 on 2017/5/24.
//  Copyright © 2017年 醉看红尘这场梦. All rights reserved.
//
import UIKit
import WebKit

class webViewController: UIViewController {

    var webView: WKWebView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        setupUI()

        // Do any additional setup after loading the view.
    }
    //MARK: - 设置webView
    private func setupUI(){
    
        webView = WKWebView(frame: self.view.frame)
        
        view.addSubview(webView)
        self.title = "热门活动"
        let url = URL(string: "http://m.ofo.so/active.html")!
        let request = URLRequest(url: url)
        
        webView.load(request)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
}
